src/api/apiClient.js

```diff
@@ -0,0 +1,79 @@
+let instance = null;
+export class ApiClient {
```

Comment:
This way is not clear to me. I would like to discuss this approach directly.

Reply:
> This way is not clear to me. I would like to discuss this approach directly.

I just use Singleton for ApiCluent class. We need just 1 instance.